models/google_perch/preprocessor.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import librosa
+import audioread
+from birdnetlib.exceptions import AudioFormatError
+
+from iSparrow import preprocessor_base as ppb
+
```

Comment:
This will be filled in in one of the next PRs, as it's not directly compatible with birdnetlib.